bitnami/thanos/templates/querier/psp.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if and .Values.querier.enabled .Values.querier.psp.create -}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ include "thanos.fullname" . }}-querier
+  labels: {{- include "thanos.labels" . | nindent 4 }}
+    app.kubernetes.io/component: querier
+spec:
+  fsGroup:
+    rule: RunAsAny
+  runAsUser:
+    ranges:
+    - max: 1001
+      min: 1001
+    rule: MustRunAs
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+  - secret
+---
```

Comment:
We usually separate ClusterRole and ClusterRoleBinding into different files (`psp-clusterrole.yaml` and `psp-clusterrolebinding.yaml` respectively). Could you make this change?

Reply:
Done